src/Nethermind/Nethermind.Evm/EvmStack.cs

```diff
@@ -72,7 +72,7 @@ public void PushBytes(in Span<byte> value)
 
         public void PushBytes(in ZeroPaddedSpan value)
         {
-            if (_tracer.IsTracingInstructions) _tracer.ReportStackPush(value.ToArray().AsSpan());
+            if (_tracer.IsTracingInstructions) _tracer.ReportStackPush(value);
```

Comment:
I think it is alright but be cautious not to pass some malleable data.

Reply:
This is just to avoid .ToArray().AsSpan() if tracer can handle default type. And cancellable can - it does nothing with the value.